katdal/visdatav4.py

```diff
@@ -82,6 +82,8 @@ def _add_sensor_alias(cache, new_name, old_name):
 VIRTUAL_SENSORS.update({'Antennas/{ant}/az': _calc_azel,
                         'Antennas/{ant}/el': _calc_azel})
 
+DEFAULT_CAL_PRODUCTS = ('L1.K', 'L1.B', 'L1.G')
```

Comment:
Should we have some consistency in the capitalisation of L1? In the stream names it is sdp_l1 (lower-case).

Reply:
A valid question... Every other place I could find has L1 capitalised (including the docs), and since we have the glorious standard of all-caps product types I thought this way the cal product would have a homogenous look at last 😛But the underlying streams tend to be (mostly) lower-case. I'll study the colour chart for the shed some more.

Reply:
The consensus is `l1` so there you go :smile: